### ISSUE TITLE
The performance on the Citeseer and BlogCatalog datasets is not good

### ISSUE THREAD

**Issue by wjwang00** [Graph-COM/GAD-NR#8]
Hello, have you tried the BlogCatalog and Citeseer datasets? I attempted to train and test on these two datasets, but the performance was very poor. If you have tried them, could you share the parameters that yielded the best results?

**Comment by AmitRoy7781** [comment]
I did not try on these dataset, you might need to tune the hyperparameters degree, self feature and neighbor feature reconstruction lambda, learning rate, batch size.
